guides/schema/types_and_fields.md

```diff
@@ -141,6 +141,45 @@ resolve ->(obj, args, ctx) {
 }
 ```
 
+## Mutations
+
+Registering a mutation root allows to define fields that can mutate your data.
+
+```ruby
+Schema = GraphQL::Schema.define do
+  mutation MutationRoot
+end
+
+MutationRoot = GraphQL::ObjectType.define do
+  name "Mutation"
+
+  field :addPost, Post do
+    description "Find dairy products matching a description"
+
+    # Use Input Types to define complex argument types
+    argument :post, PostInputType
+    resolve ->(t, args, c) {
+      title = args['post']['title']
+      description = args['post']['description']
+      Post.create(title: title, description: description)
+    }
+  end
+end
+
+PostInputType = GraphQL::InputObjectType.define do
+  name "PostInputType"
+  description "Properties for creating a Post"
+
+  input_field :title, !types.String do
```

Comment:
Can we recommend [`argument`](https://github.com/rmosolgo/graphql-ruby/blob/master/lib/graphql/input_object_type.rb#L30) instead of `input_field`? I would like to start recommending that one since it's the same as field `argument`s